Temporarily revert base::WriteFile to the behavior in older revision of
libchrome until we sort out the expected file permissions at all call sites of
base::WriteFile in Chrome OS code (crbug.com/412057).

diff --git a/base/files/file_util_posix.cc b/base/files/file_util_posix.cc
index 07c21d1..453cda3 100644
--- a/base/files/file_util_posix.cc
+++ b/base/files/file_util_posix.cc
@@ -678,7 +678,7 @@ int ReadFile(const FilePath& filename, char* data, int max_size) {
 
 int WriteFile(const FilePath& filename, const char* data, int size) {
   ThreadRestrictions::AssertIOAllowed();
-  int fd = HANDLE_EINTR(creat(filename.value().c_str(), 0640));
+  int fd = HANDLE_EINTR(creat(filename.value().c_str(), 0666));
   if (fd < 0)
     return -1;
 
